engine/card.py

```diff
@@ -22,7 +20,7 @@ def __str__(self):
     def __repr__(self):
         return str(self)
 
-    def make_public(self, information_type):
+    def make_public(self, information_type, information=None):
         if information_type == 'color':
```

Comment:
I'm aware that `information` is an unused parameter, but when a wild `Card` has its color made public it needs to be aware of what that color is to correctly pass that data to an AI. I'll take care of implementing this change in `Card` with this PR after I get feedback on `HanabiColor`.